projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py

```diff
@@ -700,6 +700,8 @@ def __helm_install_command(self, git_server_ip):
             "--set",
             "extraEnvVars.SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI=http://localhost",
             "--set",
+            "extraEnvVars.SWAGGER_UI_HOSTNAME=http://localhost:8092",
```

Comment:
nit: I discourage making changes in multiple components in the same PR. 

I prefer using stacked PRs (https://benjamincongdon.me/blog/2022/07/17/In-Praise-of-Stacked-PRs/) . But I am not insistent on it. I just recommend it :) 

I think it's better for easier reversibility , clear commit log and tracing, release notes are clearer, currently release notes would only note the change in vdk-server but not the one is control-service

Reply:
I was actually planning on doing multiple PRs at first. 

If you look in my commit history. you can see I was only making the control-plane changes. 


But then to test it I needed to make these changes. 

Then given the size of the PR it just felt right to put it in the one PR.

Reply:
> But then to test it I needed to make these changes.

Stacked PRs enabled that - https://benjamincongdon.me/blog/2022/07/17/In-Praise-of-Stacked-PRs/